versioned_docs/version-24.04/administration/centreon-ha/acceptance-guide.md

```diff
@@ -0,0 +1,712 @@
+```
+
+If you run the `crm_mon` command on node 2, you can see that node 1 is still the passive node:
+
+```text
+Cluster Summary:
+  * Stack: corosync
+  * Current DC: @CENTRAL_NODE1_NAME@ (version 2.1.2-4.el8_6.3-ada5c3b36e2) - partition with quorum
+  * Last updated: Tue Nov 8 17:27:28 2022
+  * Last change:  Tue Nov  8 17:23:19 2022 by root via crm_attribute on @CENTRAL_NODE2_NAME@
+  * 2 nodes configured
+  * 12 resource instances configured
+Node List:
+  * Online: [ @CENTRAL_NODE1_NAME@ @CENTRAL_NODE2_NAME@ ]
+Full List of Resources:
+  * Clone Set: ms_mysql-clone [ms_mysql] (promotable):
+    * ms_mysql  (ocf::heartbeat:mariadb-centreon):       Stopped @CENTRAL_NODE1_NAME@ (Monitoring)
+    * Masters: [ @CENTRAL_NODE2_NAME@ ]
+    * Stopped: [ @CENTRAL_NODE1_NAME@ ]
+  * Clone Set: php-clone [php]:
+    * Started: [ @CENTRAL_NODE1_NAME@ @CENTRAL_NODE2_NAME@ ]
+  * Clone Set: cbd_rrd-clone [cbd_rrd]:
+    * Started: [ @CENTRAL_NODE1_NAME@ @CENTRAL_NODE2_NAME@ ]
+  * Resource Group: centreon:
+    * vip       (ocf::heartbeat:IPaddr2):        Started @CENTRAL_NODE2_NAME@
+    * http      (systemd:httpd):         Started @CENTRAL_NODE2_NAME@
+    * gorgone   (systemd:gorgoned):      Started @CENTRAL_NODE2_NAME@
+    * centreon_central_sync     (systemd:centreon-central-sync):         Started @CENTRAL_NODE2_NAME@
+    * cbd_central_broker        (systemd:cbd-sql):       Started @CENTRAL_NODE2_NAME@
+    * centengine        (systemd:centengine):    Started @CENTRAL_NODE2_NAME@
+    * centreontrapd     (systemd:centreontrapd):         Started @CENTRAL_NODE2_NAME@
+    * snmptrapd (systemd:snmptrapd):     Started @CENTRAL_NODE2_NAME@
+Migration Summary:
+  * Node: @CENTRAL_NODE1_NAME@:
+    * ms_mysql: migration-threshold=1000000 fail-count=1000000 last-failure='Tue Nov  8 17:27:25 2
+022'
+Failed Resource Actions:
+  * ms_mysql_start_0 on @CENTRAL_NODE1_NAME@ 'error' (1): call=440, status='complete', exitreason='M
+ariaDB slave io has failed (1236): Got fatal error 1236 from master when reading data from binary
+log: 'Error: connecting slave', last-rc-change='Tue Nov  8 17:27:21 2022', queued=0ms, exec=4060ms
+```
+
+If you want to switch to the active node, you must do a failover.
+So, **before you do this, you must check the cluster and database replication status**.
+
+First, check the constraints:
+
+```shell
+pcs constraint
+```
+
+The command should return this:
+
+```text
+Location Constraints:
+Ordering Constraints:
+Colocation Constraints:
+  centreon with ms_mysql-clone (score:INFINITY) (rsc-role:Started) (with-rsc-role:Master)
+  ms_mysql-clone with centreon (score:INFINITY) (rsc-role:Master) (with-rsc-role:Started)
+Ticket Constraints:
+```
+
+then check the database replication
+
+```bash
+/usr/share/centreon-ha/bin/mysql-check-status.sh
+```
+
+At this time, the cluster is in degraded mode with two passive nodes.
+In this particular case, it returns the following information because the ms_mysql resource is stopped on node 1:
```

Comment:
I had not paid attention before, but we are not in the scenario where the database is on two nodes that are not the central according to this sentence. This does not match what is described earlier, though, does it? (look for  @DATABASE_NODE1_NAME@)